src/openapi/streaming/subscription.js

```diff
@@ -1,4 +1,4 @@
-/* eslint max-lines: ["error", 700] */
+/* eslint max-lines: ["error", 750] */

```

Comment:
maybe we hsould just disable this ;)

Reply:
To be honest, I would leave it be. It's good reminder to try a encapsulate some close coupled functionalities. 

However, I would happily change default to 900 from 300, globally.

In the end, lines per functions would be better indicator of 'clean code'.